pkg/kubectl/cmd/util/factory.go

```diff
@@ -353,7 +354,7 @@ func (c *clientSwaggerSchema) ValidateBytes(data []byte) error {
 	//       instead of trying everything.
 	err = getSchemaAndValidate(c.c.RESTClient, data, "api", version, c.cacheDir)
 	if err != nil && c.ec != nil {
-		errExp := getSchemaAndValidate(c.ec.RESTClient, data, "experimental", version, c.cacheDir)
+		errExp := getSchemaAndValidate(c.ec.RESTClient, data, "apis"+"/"+latest.GroupOrDie("experimental").Group, version, c.cacheDir)
```

Comment:
Sure.

Also this is the kubectl change, where else do you think we need a change?